frontend/src/metabase/query_builder/components/filters/pickers/DatePicker/DatePickerShortcuts.tsx

```diff
@@ -25,7 +25,7 @@ function getDateTimeField(field: Field, bucketing?: string) {
 
 type Option = {
   displayName: string;
-  init: (filter: any) => any[];
+  init: (filter: Filter) => any;
```

Comment:
~Not sure why you change the return type to non-array as all `init` function in this file still returns an array.~

~Also, I just realized you type the return type as `FilterExpression` here, we should make ensure they're consistent right?~

~https://github.com/metabase/metabase/pull/23464/files#diff-77360c3f5d729ad4d4fa8e125bd52a5a847d9a1732c2e34280ee393f618ffcc9R56~

#### Edited
Ok, I see you change it to any to avoid type error. Maybe you could add some comments on why this is necessary, otherwise, it won't be obvious why this change is needed.

Ideally, I'd prefer fixing the type, but I could see this would take quite an effort.

Reply:
Yeah, I think it would take a lot of refactoring to get this right all the way down. I think having a type on the function argument at least improves things